INFO:root:Program started
INFO:root:Robot set up!
INFO:root:Odometry was started, PID: 5972:
INFO:root:Initial location: 0.0, 0.0, 0.0
WARNING:root:Traceback (most recent call last):
  File "/home/pi/oscar/main.py", line 53, in main
    oscar.navigateMap([0, 0], [4, 1])
  File "/home/pi/oscar/Robot.py", line 113, in navigateMap
    arrived = gt.go_to(self, destination_coord)
  File "/home/pi/oscar/go_to4.py", line 11, in go_to
    while abs(th - th.spun) < 0.08:
AttributeError: 'numpy.float64' object has no attribute 'spun'

WARNING:root:Abruptly stopped the robot!
INFO:root:Odometry was stopped
